### PR TITLE
Fix AddonManager for raspberry-pi

### PR DESCRIPTION
**Pull request by cyr-ius**
Missing platform for raspberry-bi , fix for load addons

### CODE REVIEW

**Comment by MartijnKaijser** [comment]
@tamland ping

File: xbmc/addons/AddonManager.cpp [hunk 1]
```diff
@@ -934,7 +934,7 @@ bool CAddonMgr::PlatformSupportsAddon(const cp_plugin_info_t *plugin) const
 #if defined(TARGET_FREEBSD)
         || *platform == "freebsd"
 #endif
-        )
+        || *platform == "rbpi")
```
Comment (tamland) [inline xbmc/addons/AddonManager.cpp]: That doesn't look correct. It would make it return true for everything not freebsd. Feel free to restructure this whole section because it's really hard to follow..

**Comment by tamland** [comment]
It's really @Montellese's area, but I'm guessing it's an error from https://github.com/xbmc/xbmc/pull/7570

**Comment by Cyr-ius** [comment]
Sorry, I did not find Montlesse area, yes,this is the same problem #7570  

When I build Kodi with the parameter --with-platform = raspberry-pi
Addon The XML files are marked with the platform name: rbpi
In fact, the manager does not recognize the addon

**Comment by wsnipex** [comment]
rbpi is TARGET_RASPBERRY_PI _and_ TARGET_LINUX, so the current code should work.
Can you be more specific about the error you see?

**Comment by tamland** [comment]
@wsnipex There is no check for the "rbpi" platform at all, so TARGET_RASPBERRY_PI will only contain `if (*platform == "linux")`. Hench it would not load addons with platfrom "rbpi".  Correct? Thats how I read it at lest.

**Comment by wsnipex** [comment]
right, those ifdefs are a bit unreadable.
in this case || *platform == "rbpi" should be on L933, besides linux
